src/products/iOS_products/review_iOSstrings.md

```diff
@@ -1,14 +1,14 @@
-# Reviewing strings for a new release of Firefox for iOS
+# Reviewing strings for a new release of a Mozilla iOS product
 
-Usually, a new release of Firefox for iOS means an update to strings. If this is the case, a pull request will be made by the iOS team, in order to land these new strings in the [mozilla-l10n firefoxios-l10n repository](https://github.com/mozilla-l10n/firefoxios-l10n).
+Usually, a new release of our iOS products means an update to strings. If this is the case, a pull request will be made by the iOS team, in order to land these new strings, either in the [mozilla-l10n firefoxios-l10n repository](https://github.com/mozilla-l10n/firefoxios-l10n) or the [mozilla-l10n focusios-l10n repository](https://github.com/mozilla-l10n/focusios-l10n).
 
 At this point, the PR is reviewed by an l10n-driver - most often by the PM in charge of the project.
 
 Let’s go over some of the steps needed over time in order to review correctly strings for a new release.
 
 ## Reviewing the PR
 
-When you’re getting close to the end of the [Firefox for iOS l10n cycle](https://wiki.mozilla.org/Firefox_for_iOS_Train_Schedule), the iOS team will make a pull request in order to land new strings for the upcoming release.
+Let's take examples from Firefox for iOS - in fact, these apply to other iOS products as well, and they also follow the same train schedule. When you’re getting close to the end of the [iOS l10n cycle](https://docs.google.com/spreadsheets/d/1hER_w7pOsWSjeBaUqMd_9ClYCCGlwc4jaFYpcBvFE8g/edit#gid=1960046531), the iOS team will make a pull request in order to land new strings for the upcoming release.
```

Comment:
Is it just me or "Let's take examples" sounds strange? Not sure if it's an adapted French expression.
```suggestion
Let's consider Firefox for iOS as an example - in fact, these apply to other iOS products as well, and they also follow the same train schedule. When you’re getting close to the end of the [iOS l10n cycle](https://docs.google.com/spreadsheets/d/1hER_w7pOsWSjeBaUqMd_9ClYCCGlwc4jaFYpcBvFE8g/edit#gid=1960046531), the iOS team will make a pull request in order to land new strings for the upcoming release.
```